src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="comment-container paddingT20" id="wi-comment" style="clear:both">
+    <div class="comment-content" id="wi-comment-content">
+        <!-- Refactoring needed to toogle view -->
+        <div *ngIf="true" class="width100">
+            <div class="width100  paddingL20 paddingR20" *ngIf="true">
+                <div *ngIf="loggedIn" class="paddingB20">
+                    <textarea #commentText rows="2" type="text" id="exampleDesc" class="form-control width100" [(ngModel)]="comment.attributes.body"
```

Comment:
This is the 'create comment text area'. There will be only one, but yea, commentText is a bad name. Suggestions?